inc/woocommerce/storefront-woocommerce-template-functions.php

```diff
@@ -679,7 +679,7 @@ function storefront_handheld_footer_bar_search() {
 	 */
 	function storefront_handheld_footer_bar_cart_link() {
 		?>
-			<a class="footer-cart-contents" href="<?php echo esc_url( wc_get_cart_url() ); ?>" title="<?php esc_attr_e( 'View your shopping cart', 'storefront' ); ?>">
+			<a class="footer-cart-contents" href="<?php echo esc_url( wc_get_cart_url() ); ?>"><?php esc_attr_e( 'Cart', 'woocommerce' ); ?>
```

Comment:
We need to keep the same translation domain name and the escaping function should be now `esc_html_e()` instead of `esc_attr_e()` because it's no longer an attribute.
```suggestion
			<a class="footer-cart-contents" href="<?php echo esc_url( wc_get_cart_url() ); ?>"><?php esc_html_e( 'Cart', 'storefront' ); ?>
```